### PR TITLE
Docs: add cards RTL example

### LINKED ISSUE #1549: Reintroduce at least an example page for RTL
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

RTL examples are all coming from Bootstrap and those examples have been deleted.

We should at least have one example in RTL and reintroduce the diff of this PR: https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1542

### Motivation and context

f

### PR DESCRIPTION
**Pull request by hannahiss**
### Related issues

closes #1549 

### Description
- Add category in site/data/examples.yml
- Add cards RTL example file cards-rtl/index.html
- Update _index.md as in Bootstrap https://getbootstrap.com/docs/5.2/examples/ in particular warning about RTL that is still experimental
- Add 2 images in site/static/docs/5.2/assets/img/examples

### Motivation & Context

Need of an RTL example

### Types of change

- New feature (non-breaking change which adds functionality)

### Live previews

* https://deploy-preview-1603--boosted.netlify.app/

### Checklist

- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)
- [x] My code follows the code style of the project _(using `npm run lint`)_
- [x] My change introduces changes to the documentation
- [x] My change introduces changes to the migration guide
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

### CODE REVIEW

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
1 thing to tackle before approval!

**Review by louismaximepiton — APPROVED** [review]
LGTM!

**Comment by julien-deramond** [comment]
Please add something in the migration guide to explain that we now have RTL examples and a new RTL example page for cards.

**Review by julien-deramond — APPROVED** [review]
LGTM 🚀 
Made few minor changes in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1603/commits/4f4bb9d90d14fb0158f0008cbf179b2a0e597290, https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1603/commits/81a58b6d002ecbc27b0a7a190d5df4408e159ca1 and https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1603/commits/1be081e53d9c66b9aae62c7f570e4b9e5a3ca322

File: site/content/docs/5.2/examples/cards-rtl/index.html [hunk 1]
```diff
@@ -0,0 +1,572 @@
...
+
+      <div class="col m-0 py-2">
+        <div class="card border-0">
+          {{< placeholder width="100%" height="162" class="card-img-top" text="تعليق على الصورة" >}}
+          <div class="card-body bg-dark">
+            <h4 class="card-عنوان">عنوان</h4>
+            <h6 class="card-text">شرح</h6>
+          </div>
+          <div class="card-footer bg-dark text-light pt-2">
+            <a class="btn btn-secondary" aria-label="وصف الزر" title="وصف الزر">زر</a>
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/content/docs/5.2/examples/cards-rtl/index.html:102]: The page rendering is OK in the browsers. The text is well-displayed in the IDE as well. So yeah, it must be an issue on GitHub side.

**Pull request merged**